### ISSUE TITLE
change the last layers to 

### ISSUE THREAD

**Issue by wenxin06133** [jlianglab/BenchmarkTransformers#5]
if i wnat to change sofmax to sigmoid in the last layer , how can i find the structure i used.

**Comment by Mda233** [comment]
In this repo, the output from the model is the pre-logits without activation layer. You can add a softmax or sigmoid layer after 

 `outputs = model(samples)`, i.e., `outputs = torch.nn.Sigmoid(outputs)`.

**Comment by Mda233** [comment]
BTW, when you have a `outputs = torch.nn.Sigmoid(outputs)`, please make sure you are using BCELoss instead of BCEWithLogitsLoss.

**Comment by wenxin06133** [comment]
Thanks for reply my question.
But what is the meaning of the p_test?
Is that the probability of each class?

Mda233 ***@***.***>於 2023年5月4日 週四，下午1:50寫道：

**Comment by Mda233** [comment]
> Thanks for reply my question. But what is the meaning of the p_test? Is that the probability of each class? 

Yes,  p_test is the prediction probability of each class from the mode.

**Comment by wenxin06133** [comment]
Which file can I put this code in?
outputs = torch.nn.Sigmoid(outputs)

Mda233 ***@***.***>於 2023年5月4日 週四，下午1:58寫道：

**Comment by Mda233** [comment]
> Which file can I put this code in? outputs = torch.nn.Sigmoid(outputs)

In trainer.py, the train_one_epoch method.
